### ISSUE TITLE
wrong generated COBOL in 1.0.16

### ISSUE THREAD

**Issue by GitMensch** [mridoni/gixsql#83]
I'm a bit confused because in one environment everything seems to work, while in another it doesn't but generate SQL definitions like the following, which of course raise a compiler error then:

```cobol
GIXSQL 01  SQ0031.
GIXSQL     02  FILLER PIC X(0030) VALUE "DELETE FROM POS WHERE OID = $1".
GIXSQL     02  FILLER PIC X(1) VALUE X"00".

...

GIXSQL*    EXEC SQL
GIXSQL*        DELETE FROM POS
GIXSQL*        WHERE OID = :OID
GIXSQL*    END-EXEC.
GIXSQL     CALL "GIXSQLStartSQL"
GIXSQL     END-CALL
GIXSQL     CALL "GIXSQLSetSQLParams" USING
GIXSQL         BY VALUE 22
GIXSQL         BY VALUE 9
GIXSQL         BY VALUE 0
GIXSQL         BY VALUE 0
GIXSQL         BY REFERENCE OID
GIXSQL     END-CALL
GIXSQL     CALL "GIXSQLExecParams" USING
GIXSQL         BY REFERENCE SQLCA
GIXSQL         BY REFERENCE x"00"
GIXSQL         BY VALUE 0
GIXSQL         BY REFERENCE SQ0031
GIXSQL         BY VALUE 1
GIXSQL     END-CALL
GIXSQL     CALL "GIXSQLEndSQL"
GIXSQL     END-CALL.

```

Any idea what is happening / broken in this environment? Or is it gixpp to be blamed with a command that has exactly this length / layout?

**Comment by mridoni** [comment]
Is by any chance the definition in the first line > 72 bytes long? Yesterday, while working on another issue I found something similar. I thought it was a corner case so I took a note and went on with what I was doing, but maybe it deserves a more accurate investigation.

**Comment by GitMensch** [comment]
The generated definition is 73 bytes long, yes.

**Comment by mridoni** [comment]
Ok, I reproduced it

**Comment by mridoni** [comment]
Fixed in the internal repository, will be in in v1.0.17

**Comment by GitMensch** [comment]
Fixed in 1.0.17
